### PR TITLE
moved monitor scripts 

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
This pull request depends on https://github.com/ipa320/cob_common/pull/130

File: cob_monitoring/CMakeLists.txt [hunk 1]
```diff
@@ -5,10 +5,10 @@ project(cob_monitoring)
 find_package(catkin REQUIRED COMPONENTS)
 
 catkin_package(
-    CATKIN_DEPENDS cob_msgs cob_script_server
+    CATKIN_DEPENDS cob_msgs cob_script_server std_msgs
```
Comment (floweisshardt) [inline cob_monitoring/CMakeLists.txt]: do we still need a dependency to std_msgs? I think all messages are in cob_msgs now, see https://github.com/ipa320/cob_common/pull/130

File: cob_monitoring/package.xml [hunk 2]
```diff
@@ -8,13 +8,18 @@
...
   <author email="fmw@ipa.fhg.de">Florian Weisshardt</author>
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <build_depend>sysstat</build_depend>
+  <build_depend>ipmitool</build_depend>
+
   <run_depend>cob_msgs</run_depend>
   <run_depend>cob_script_server</run_depend>
+  <run_depend>std_msgs</run_depend>
```
Comment (floweisshardt) [inline cob_monitoring/package.xml]: can be remove, see above

File: cob_monitoring/package.xml [hunk 3]
```diff
@@ -8,11 +8,15 @@
...
-   <maintainer email="fmw@ipa.fhg.de">Florian Weisshardt</maintainer>
+  <maintainer email="fmw@ipa.fhg.de">Florian Weisshardt</maintainer>
+  <maintainer email="nhg@ipa.fhg.de">Nadia Hammoudeh</maintainer>
   <author email="fmw@ipa.fhg.de">Florian Weisshardt</author>
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <build_depend>sysstat</build_depend>
+  <build_depend>ipmitool</build_depend>
+
```
Comment (fmessmer) [inline cob_monitoring/package.xml]: Where do these dependencies come from?
Are they needed as `build_depend` or `run_depend` or both?

**Comment by fmessmer** [comment]
LGTM
Are the new scripts tested on hardware?

**Comment by floweisshardt** [comment]
visual review: OK

File: cob_monitoring/package.xml (hunk 3 above)
Comment (NadiaHG) [inline cob_monitoring/package.xml]: cpu_monitor reports the cpu statistics using iostat (from sysstat) , mpstat (from sysstat) and ipmitool.
Added also python-mechanize (web browsing in python) , needed to check the ddwrt status

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**